tests/harambe_contract.py

```
@@ -0,0 +1,388 @@
+    """
+    import importlib
+
+    found = {}
+    for module_path in harambe_modules():
+        relative = module_path.relative_to(HARAMBE_DIR)
+        dotted = "harambe_scrapers." + ".".join(relative.with_suffix("").parts)
+        try:
+            module = importlib.import_module(dotted)
+        except Exception:  # noqa: BLE001 - a module needing a browser to import
+            # is not a naming defect; it is simply out of reach here.
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)